tests/integration/src/main/groovy/com/datadog/appsec/php/docker/AppSecContainer.groovy

```diff
@@ -100,7 +100,7 @@ class AppSecContainer<SELF extends AppSecContainer<SELF>> extends GenericContain
         if (doWithConn) {
             doWithConn.call(conn)
         }
-        if (conn.doOutput) {
+        if (conn.getRequestMethod() != 'POST' && conn.doOutput) {
```

Comment:
Can you clarify why we don't close the output stream in POST?

Reply:
I've just removed the conditional and its content from here. What was that nobody was setting the output to true before this PR. No I am, on the Closure I need to check for the http response code and that was closing the output and by the time this conditional tried to close it, it was already closed.